dataactcore/config_example.yml

```diff
@@ -33,63 +43,112 @@ broker:
     # File D1 API
     # Uses "{}" because it requires string formatting to insert query parameters
     # If the URL is provided, the file name provided will be used when uploading to S3
-    award_procurement_url: https://data-usas.uat.workplace.gov/api/contracts?CGAC={}&date_signed_beginning={}&date_signed_end={}&callback={}
+    award_procurement_url: https://award_procurement_url.gov
     award_procurement_file_name: d1_data.csv
 
     # File D2 API
     # If the URL is provided, the file name provided will be used when uploading to S3
-    award_url: https://data-usas.uat.workplace.gov/api/assistance?CGAC={}&date_signed_beginning={}&date_signed_end={}&callback={}
+    award_url: https://award_url.gov
     award_file_name: d2_data.csv
 
-    d_file_storage_path: /full/path/to/d/file/storage
+    d_file_storage_path: /data-act/backend/data-act-broker-backend/tmp/
 
     # File E
-    awardee_attributes_url: https://sample.gov
     awardee_attributes_file_name: awardee_data.csv
     executive_compensation_url: https://sample.gov
     executive_compensation_file_name: executive_compensation_data.csv
 
     # File F
-    sub_award_url: https://sample.gov
     sub_award_file_name: sub_award_data.csv
 
-    # Include folder and filename of RSS file if running on AWS, if running locally the RSS folder and file should be in
-    # the broker_files directory specified above
-    rss_folder: rss
-    rss_file: rss_file.xls
-    
+    # Connection info for fetching data related to File F
+    fsrs:
+        procurement_service:
+            wsdl: 'https://www.fsrs.gov/ws/ffata_data_api.php?wsdl'
+            username: 'ffataexport'
+            password: 'password'
+        grant_service:
+            wsdl: 'https://www.fsrs.gov/ws/ffata_grants_data_api.php?wsdl'
+            username: 'ffataexport'
+            password: 'password'
```

Comment:
For this section above here, do we want to keep the real data? @michaeldhess ?

Reply:
I'm thinking no. It looks like the real pw isn't in there, but I'd remove the real username as well (although it's already exposed in the Git history).